<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>try catch 구문사용</title>
</head>
<body>
    <script>
        function test(){
        try{
            console.log(date + 'a입니다.')
            throw new Error('난너무잘난듯!.')
        }
        catch(e){ //e로 매게변수 사용
            console.log('catch 에서 받은 에러 : '+e) //에러 내용출력
            return  //finally를 내용을 제외하고 무조건 종료
        }
        finally{
            console.log('함수 실행을 마칩니다.')  //무조건 실행합니다.
        }    
    }
    </script>
</body>
</html>